projects/storefrontapp/src/app/app.module.ts

```diff
@@ -45,6 +50,8 @@ if (!environment.production) {
         }
       }
     }),
+
+    ConfigModule.withConfigFactory(defaultCmsContentConfig),
```

Comment:
Do we want to keep this here like this?
What about wrapping it into a module, so instead of `ConfigModule.withConfigFactory(defaultCmsContentConfig),` it would be something like `DefaultCmsContentModule`.

Reply:
Agreed, but I'll keep that for the next sprint. It will take a bit of structure for various elements of the default slots.

I have however removed the `ignoreBackend: false` for the `cartPage` for now.